pyproject.toml

```
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["flit"]
-build-backend = "flit.buildapi"
+requires = ["flit_core"]
```

```suggestion
requires = ["flit_core  >=2,<4"]
```

[See the docs](https://flit.pypa.io/en/stable/pyproject_toml.html#build-system-section). I encourage all packages to specify an upper bound on flit_core, so that I can make changes in future versions without breaking lots of packages.